### ISSUE TITLE
PlantUML image unencrypted http request makes website "insecure"

### ISSUE THREAD

**Issue by UnlimitedCookies** [Requarks/wiki#1256]
**Describe the bug**
PlantUML requests the image via unencrypted HTTP and therefore shows a warning message to the user that the site is insecure.

This is requested:
```
<img src="http://www.plantuml.com/plantuml/svg/U9nLZy4AWp0CXkzvYi3Ele18iKw4NJV6pg7c8DRMfN7Flpg5QGu9uV_oywTEIgBBv67ESt33J47ncLW0rew5GLjY6rzHHY2dmuUKqUAbwvfS07P7Ra5P0dbqvFs9R4d61_afXfrkIgmgTC8PkFDxuQHepFyckpPORBQxVlt1FVHye0wBtPpNo6lQm6SHQWvz_l8BXQPBk000" alt="uml diagram" class="uml-diagram">
```

**To Reproduce**
Go to: https://docs.requarks.io/releases
See read warning message.
**Expected behavior**
Shiny encrypted picture from PlantUML

**Comment by UnlimitedCookies** [comment]
This might have been solved by #1223 but the wiki.js docs website still uses an HTTP image on its release site - So there's still an issue apparently.

**Comment by NGPixel** [comment]
The fix only affects new installations. Existing installations must be fixed via the DB or wait the next release which will allow to save the rendering configuration from the admin area.

The releases page on the wiki.js docs website has been fixed.
